### PR TITLE
New style

### PR DESCRIPTION
**Pull request by boyd-nguyen**
- Add ADO logo
- Change font
- Restyle navbar
- Change heading 2 to heading 1

This is a rough new layout, and there will be more work on the design of specific elements.

### CODE REVIEW

File: Australian Digital Observatory/assets/static/style_custom.css [hunk 1]
```diff
@@ -59,28 +70,13 @@ h3, h4 {
 
 .accordion-header * {
```
Comment (Quiet27) [inline Australian Digital Observatory/assets/static/style_custom.css:71]: Does the font for the accordion need to be changed as well?

**Review by Quiet27 — COMMENTED** [review]
@boyd-nguyen I really like the new style! I only noticed one thing - the accordion font seemed to be the same as before (but I'm not 100% sure because I didn't have the new and old open side-by-side). Should the font be updated for the accordion?

File: Australian Digital Observatory/assets/static/style_custom.css (hunk 1 above)
Comment (boyd-nguyen) [reply Australian Digital Observatory/assets/static/style_custom.css:71]: Hi Alice, I've inspected the accordions and I believe it's the same font. I do understand why you might see the font as different from the rest of the body text. I've changed the boldness and size of the accordion text. Hopefully now they look more congruent, but let me know if you're still seeing a different font.

**Review by Quiet27 — APPROVED** [review]
Nice job, I don't have any further feedback 😸

**Pull request merged**